Kudu.Core/Deployment/Generator/OryxBuilder.cs

```diff
@@ -93,6 +93,19 @@ private static void PreOryxBuild(DeploymentContext context)
                     FileSystemHelpers.DeleteDirectorySafe(pythonPackagesDir);
                 }
             }
+
+            if (args.Flags != BuildOptimizationsFlags.DeploymentV2)
+            {
+                // Cleanup site packages of old deployment artifacts if present
+                string sitePackages = "/home/data/SitePackages";
+                string[] filesInSitePackages = { "packagepath.txt", "packagename.txt" };
+
+                foreach (string fileName in filesInSitePackages)
+                {
+                    string filePath = Path.Combine(sitePackages, fileName);
+                    FileSystemHelpers.DeleteFileSafe(filePath);
+                }
+            }
```

Comment:
We should clear up packagename.txt and packagepath.txt for all scenarios where the current build  is not going to produce it. (Not just deploymentv2).

Reply:
These artifacts will be produced only by DeploymentV2 and ExpressBuild flags. If these flags aren't set I have cleared out the packagename.txt and packagepath.txt.

Reply:
`packagename.txt` could also be produced by `RunFromPackage`. A point to consider: if the build fails - this would remove the link for last running build and could also affect runtime. We should instead do this in the PostBuild step. There we could simply check if the file exists and delete it before taking the separate path for `ExpressBuild`, `DeploymentV2` etc. which would recreate this file if needed.

Reply:
Have made changes to remove these artifacts after the build is done. Also, the code will remove these artifacts if they are present irrespective of the BUILD_FLAGS configured. They get auto-generated after this if DeploymentV2 or ExpressBuild is set.